app/ygnmi/cmd/generator/generator.go

```diff
@@ -57,6 +57,7 @@ func New() *cobra.Command {
 	generator.Flags().Int("structs_split_files_count", 1, "The number of files to split the generated schema structs into.")
 	generator.Flags().Int("pathstructs_split_files_count", 1, "The number of files to split the generated path structs into.")
 	generator.Flags().Bool("ignore_deviate_notsupported", false, "If set to true, 'deviate not-supported' YANG statements are ignored, thus target nodes are retained in the generated code.")
+	generator.Flags().Bool("ignore_unsupported", false, "If set to true, YANG statements unsupported by ygot are ignored.")
```

Comment:
why is this flag needed now?
can this be always true or false?

Reply:
This is to support use cases where there are unsupported statements.

I don't think we should be opinionated about this one: users should be aware that their YANG has unsupported statements and intentionally turn this flag on, rather than find out later that some leaves are not supported.